packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -64,6 +67,10 @@ export default class BotonicPluginFlowBuilder implements Plugin {
     this.getAccessToken = resolveGetAccessToken(options)
     this.trackEvent = options.trackEvent
     this.getKnowledgeBaseResponse = options.getKnowledgeBaseResponse
+    this.smartIntentsConfig = {
+      ...(options.smartIntentsConfig || {}),
+      use_latest: jsonVersion === FlowBuilderJSONVersion.LATEST,
+    }
```

Comment:
You don't need to do || {}, if you deconstruct smartIntentsConfig and it is undefined nothing happens.
By default we don't set `num_smart_intents_to_use: 10`

```suggestion
    this.smartIntentsConfig = {
      ...options.smartIntentsConfig,
      use_latest: jsonVersion === FlowBuilderJSONVersion.LATEST,
      num_smart_intents_to_use: 10
    }
```

Reply:
destructuring ...undefined will return a runtime error:
```
> a = undefined
undefined
> ...a
...a
^^^

Uncaught SyntaxError: Unexpected token '...'
```
num_smart_intents_to_use doesn't need to be specified in the api call

Reply:
simplified as:
```
this.smartIntentsConfig = {
      ...options?.smartIntentsConfig,
      use_latest: jsonVersion === FlowBuilderJSONVersion.LATEST,
    }
```